### PR TITLE
Assignment7+8 + @NazariyPogo

### PR DESCRIPTION
**Pull request by Ashrom0**
Partner = @NazariyPogo -Nazariy Pogoriliy

### CODE REVIEW

**Comment by ZackWard-student** [comment]
Review in progress. 3/9/22 7pm

**Comment by ZackWard-student** [comment]
**General**

The project should look like the image below ✔

The project should have an index.html file ✔

With an index.js file in a js folder ✔

And a site.css file in a css folder (along with other css files)✔

**There should be a \<head>**

With a favicon ✔

And a title ✔

The \<body> should use the middle 70% of the page ❌

> Looks like it is set to 100% with a small margin on the body

**Menu Bar**

The page should have a title/menu bar ✔

Use Font Awesome for the hamburger icon ✔

When clicked, the menu should appear under it ✔

> Note: When clicking the hamburger icon the button does not work

The menu should have four buttons with creative names (the buttons don't need to do anything) ✔

**Create a card with \<div>s and CSS**

Create a card from \<div>s ✔

The card should have a header, body, and footer ✔

The CSS for this should be in the CSS file and work easily for another card. ✔

**Show jokes on the card**
Use the joke API from the first assignment to display a joke on a card (https://v2.jokeapi.dev/joke/Programming) ✔

Use Axios from the CDN for the API call. ✔

If the joke API returns an error display a message to try again in a few moments ❌

> The program catches the error and logs it, but does nothing else with it.

The punch line should not show immediately, but appear after 4 seconds ✔

There should be a button to get another joke ✔

There should be \<a> tag that links to the joke api documentation page ❌

`<43 button onclick="apiPage()">Joke API</button>`
> Not marked with an \<a> tag

This \<a> should be manually styled as a button ❌

> It is a button! not an \<a> tag.

Buttons should have a background that is a gradient ✔

All buttons (even the \<a> that you made into a button) should have 
a hover effect ❌

> Missing the hover effect on the Joke API button

**Extra Credit
Do the following:**

Make the buttons in the menu do something creative ❌

Rather than just making the menu appear, have it fade in and slide down. ❌

Have the page jump to 100% width on a smaller screen and 50% on a larger screen with 70% on a regular screen. ❌

**Fundamentals**
Ensure that:
This is built with raw HTML, CSS, and JavaScript, no external libraries aside from Axios and a font library ✔

The index.html file should work when opened from a file explorer ✔

Keep all files in appropriate folders ✔

Have warning free HTML, CSS, and JavaScript ✔

Use as little css as possible ✔

No inline CSS ✔

Validate HTML with: https://validator.w3.org/nu/#textarea (or similar) ✔

Validate CSS with: https://jigsaw.w3.org/css-validator/#validate_by_input (or similar) ✔

No errors when loading the page ✔

Choose simplicity over complexity ✔

**Comment by Aedyn01** [comment]
The webpage is looking decent so far, but you guys are missing some minor details. The banner and body of the website should take up 70% of the screen and be centered, and your menu button doesn't seem to be working properly, it only works when the word "Menu" is clicked, and clicking on the hamburger icon does not do anything. Clicking to the right of the icon also causes the menu dropdown to appear, so I would look into how you made that button and verify that it is the correct side and/or there aren't any invisible overlapping's happening.

File: Web/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <link rel="icon" type="image/png" sizes="32x32" href="favicon_io/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="favicon_io/favicon-16x16.png">
+    <link rel="manifest" href="favicon_io/site.webmanifest">
+</head>
+
+<body>
+    <div class="top">
+        <div class="dropdown">
+            <button class="dropbtn" onclick="DropdownFunction()">Menu
+                <i class="fa-solid fa-bars"></i>
```
Comment (mmwoodfo) [inline Web/index.html:18]: Clicking this icon does not seem to open your menu.

**Pull request closed without merging**